docs/content/stable/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md

```diff
@@ -24,9 +24,9 @@ You should perform these actions in a specific order, depending on whether perfo
 | 1. CREATE TABLE | 2. CREATE TABLE | 3. Add the table to replication |
 | 2. DROP TABLE   | 3. DROP TABLE   | 1. Remove the table from replication. |
 | 1. CREATE INDEX | 2. CREATE INDEX | 3. [Resynchronize](#resynchronize-yba) |
-| 1. DROP INDEX   | 2. DROP INDEX   | 3. [Resynchronize](#resynchronize-yba) |
+| 2. DROP INDEX   | 1. DROP INDEX   | 3. [Resynchronize](#resynchronize-yba) |
 | 1. CREATE TABLE foo PARTITION OF bar | 2. CREATE TABLE foo PARTITION OF bar | 3. Add the table to replication |
-| 1. ALTER TABLE or INDEX | 2. ALTER TABLE or INDEX | No changes needed |
+| 2. ALTER TABLE or INDEX | 1. ALTER TABLE or INDEX | No changes needed |
 
 In addition, keep in mind the following:
```

Comment:
Update line 34 as well

Reply:
Removed that line because it is covered by the table now